ds_autotuning_prototype/hf_trainer_api/README.md

```diff
@@ -0,0 +1,69 @@
+# Vision Transformer with HuggingFace Trainer API and Determined
```

Comment:
Shouldn't this file be in the `image_classification` directory if I'm reading this right?

Reply:
did a first pass at updating that readme